src/parser/Statement.ts

```diff
@@ -1604,7 +1604,7 @@ export class ClassMethodStatement extends FunctionStatement {
                 }
             }
         });
-        const walkOptions: WalkOptions = { walkMode: WalkMode.visitExpressions };
```

Comment:
@TwitchBronBron I think the problem is that `visitExpressions` should also include the `walkStatements` flag.

Reply:
Ah, yeah I think you're right. The recursive one already does exactly that.

@georgejecook can you please undo this line, and instead change the definition of `visitExpressions` to include `InternalWalkOptions.walkStatements`

https://github.com/rokucommunity/brighterscript/blob/1a84bb55f786ceb8bfc218011556e669227a72fe/src/astUtils/visitors.ts#L183

Reply:
@elsassph I made the fix we discussed (adding walkStatements to visitExpressions). That fixed George's issue, however one of the visitors tests was failing. I'm not overly familiar with that test to know how to properly validate it, so could you give it a look and see why it's failing?

Reply:
uh odd

Reply:
Ok that's because this test expects `visitExpressions` to NOT walk statements :D
I'll fix that.